python_bindings/tests/end-to-python-end-test/end_utils.py

```diff
@@ -0,0 +1,110 @@
+from typing import NamedTuple, Tuple
+
+from carcassonne_engine import GameEngine, SerializedGame
+from carcassonne_engine._bindings.elements import MeepleType
+from carcassonne_engine._bindings.feature import Type as FeatureType
+from carcassonne_engine._bindings.side import Side
+from carcassonne_engine.placed_tile import PlacedTile, Position, Tile
+from carcassonne_engine.requests import (
+    GetLegalMovesRequest,
+    MoveWithState,
+    PlayTurnRequest,
+)
+
+
+class TurnParams(NamedTuple):
+    pos: Position
+    tile: Tile
+    meepleType: MeepleType
+    side: Side
+    featureType: FeatureType
+
+
+def find_meeple_in_placed_tile(ptile: PlacedTile):
+    for feature in ptile._go_obj.Features:
+        if feature.Meeple.Type != 0:
+            print(f"sides: {feature.Sides}, feat type: {feature.FeatureType}")
+            return feature.Meeple
+    print("NONE")
+    return None
```

Comment:
This function is never used
(also, I'd remove `print("NONE")`)

Reply:
removed ( I used for the debug, because I had trouble checking where is meeple on the tile)